docs/modules/data/protobuf.md

```diff
@@ -0,0 +1,361 @@
+# Protocol Documentation
+<a name="top"></a>
+
+## Table of Contents
```

Comment:
@clevinson My idea here was to have a `Modules` section on the left, and in each module subsection, documentations about that module.

Let me know if you had something else in mind.